src/elements/content-sidebar/SidebarLoadingError.scss

```diff
@@ -4,4 +4,8 @@
     .bcs-loading-error {
         width: $sidebarContentWidth;
```

Comment:
Minor, but I wonder if we could remove these explicit widths with some clever flexbox styles. Not necessary for merge.